findCircle.py

```diff
@@ -72,13 +79,14 @@
 
 
 sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-sock.bind(("127.0.0.1", 8888))
+if (DEBUG == False):
+    sock.bind(("10.8.88.2", 8888))
```

Comment:
Shouldnt this bind be an address of 0.0.0.0?

Reply:
It should be `localhost` if I'm not mistaken. Though you can specify the exact IP it shouldn't be `10.8.88.2` since that's **always** the IP of the Rio.